spec/models/course_enrichment_spec.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: course_enrichment
+#
+#  id                           :integer          not null, primary key
+#  created_by_user_id           :integer
+#  created_at                   :datetime         not null
+#  provider_code                :text             not null
+#  json_data                    :jsonb
+#  last_published_timestamp_utc :datetime
+#  status                       :integer          not null
+#  ucas_course_code             :text             not null
+#  updated_by_user_id           :integer
+#  updated_at                   :datetime         not null
+#
+
+require 'rails_helper'
+
+describe CourseEnrichment, type: :model do
+  context 'initial draft' do
```

Comment:
I feel like these contexts don't read very well. I think it would be clearer if it was "when the status is initial draft" or something like that?

Reply:
fixed